.github/workflows/docker-build.yml

```diff
@@ -161,3 +161,26 @@ jobs:
       AWS_ROLE_PUBLISH_ARN: ${{ secrets.AWS_OIDC_IAM_ROLE_BUILD_PUBLISH_DEVELOP_PR }}
       AWS_ROLE_GATI_ARN: ${{ secrets.AWS_OIDC_GLOBAL_READ_ONLY_TOKEN_ISSUER_ROLE_ARN }}
       AWS_LAMBDA_GATI_URL: ${{ secrets.AWS_INFRA_RELENG_TOKEN_ISSUER_LAMBDA_URL }}
+
+  prepare-nightly-image-tag:
```

Comment:
We already build nightly images in this workflow.

Reply:
yes, this one only prepares the tag name

Reply:
We should have these tags already?

```
❯ aws ecr describe-images --repository-name chainlink | grep nightly
                "nightly-20250516"
                "nightly-20250516-plugins-amd64"
                "nightly-20250516-amd64"
                "nightly-20250516-plugins-arm64"
                "nightly-20250516-plugins"
                "nightly-20250516-arm64"
```

Looks like they get pruned pretty quickly which we can adjust if needed.